Documentation.md

```diff
@@ -902,19 +888,361 @@ Here is a full conversation example :
 | createdAt | Creation date of the conversation | DateTime | ✓ | ISO-8601 |
 | updateAt | Date of the last message received | DateTime | ✓ | ISO-8601 |
 
+#### Conversation objects
+
+<br />
+
+Several kinds of payloads can be used within your bot replies in order to enrich your responses. You will find in this section information about every type of content you can send with your iAdvize bot.
+
+
+##### Sending a simple message 
+
+<br />
+
+A text payload is a simple text message.
+
+<pre class="prettyprint lang-js">
+{
+    "contentType": "text",
+    "value": "Hi, i am a simple message."
+}
+</pre>
+
+| Field | Description | Values | Constraints |
+| --- | --- | --- | --- |
+| contentType | Type of the message’s content | String | `text` |
+| value | Textual content of the message | String | |
+
+<br />
+
+##### Quick reply : multiple choice question
+
+<br />
+
+A quick reply is used for offering several choices to a visitor. Each choice needs to be specified in the "quickReplies" field  of a [reply](#conversation-flow-endpoints). The answer sent by the visitor to the multiple choice question can only contain text. There is no maximum number of quick replies you can display. However we recommend not to use more than 3 quick replies for a single question.
+
+<br />
+
+<img src="./assets/images/example-quick-replies.png" alt="Example of a text payload reply with two quick replies" width="300" height="260" style="margin: auto;">
+<p align="center"><em>Quick replies example (Text payload reply with two quick replies)</em></p>
+
+<br />
+
+<pre class="prettyprint lang-js">
+{
+    "contentType": "text/quick-reply",
+    "value": "Yes",
+    "idQuickReply": "1ef5145b-a9b6-4e86-8743-b6e3b4026b2c"
+}
+</pre>
+
+<br />
+
+| Field | Description | Values | Constraints |
+| --- | --- | --- | --- |
+| contentType | Type of the message’s content | String | `text/quick-reply` |
+| value | Textual content of the quick-reply | String | |
+| idQuickReply | id of the quick reply | String | UUID |
+
+<br />
+
+##### Generic card : Sending rich-content
+
+<br />
+
+A generic card is a payload you can use to send a more structured message. It always contains at least one link and can be used to help a visitor to navigate on a website by redirecting him to specific pages. You can specify multiple links on a single generic card. Generic card can also include a title, a description and an image. This help give context to the visitor about the links you are sending.
+
+<br />
+
+<img src="./assets/images/example-generic-card-with-title-text-and-picture.png" alt="Example of a generic card with title, text and picture set" width="300" height="260" style="margin: auto;">
+<p align="center"><em>A generic card with a title, a text, an image and a single link.</em></p>
+
+<br />
+
+<img src="./assets/images/example-generic-card-with-several-links.png" alt="Example of a generic card with several links" width="300" height="260" style="margin: auto;">
```

Comment:
Il faut le faire en Markdown svp.

Reply:
On a que les <PRE> pour formater le code JSON, je ne sais pas pourquoi exactement mais il faut faire le reste en Markdown car on génère le HTML après.

Reply:
C'est toujours du markdown en insérant une balise img : https://www.xaprb.com/blog/how-to-style-images-with-markdown/. Sinon faudrait rajouter un fichier CSS pour que je puisse configurer les images